packages/react/src/components/DetailsList/DetailsList.base.tsx

```diff
@@ -675,7 +675,7 @@ const DetailsListInner: React.ComponentType<IDetailsListInnerProps> = (
         // ariaLabel is a legacy prop that used to be applied on the root node, which has poor AT support
         // it is now treated as a fallback to ariaLabelForGrid for legacy support
         aria-label={ariaLabelForGrid || ariaLabel}
-        aria-rowcount={isPlaceholderData ? -1 : rowCount}
+        aria-rowcount={isPlaceholderData ? 0 : rowCount}
```

Comment:
The description for this change states, "...the column is saying row 1 of zero...instead of row 1 of 1."

How is the column saying this? Via a screen reader like Narrator or NVDA?

[-1 is a valid value for `aria-rowcount`](https://w3c.github.io/aria/#aria-rowcount) and might be the correct thing here:

> If the total number of rows is unknown, authors MUST set the value of [aria-rowcount](https://w3c.github.io/aria/#aria-rowcount) to -1 to indicate that the value should not be calculated by the user agent.

[`aria-busy`](https://w3c.github.io/aria/#aria-busy) will also be set to `true` in this case, indicating that `isPlaceholderData` means that `DetailsList` is being modified in some way (e.g., loading data) which leads me to think that `aria-rowcount="-1"` is the correct value in this instance and that, assuming this is being read by a screen reader, the issue is there.

Reply:
So I did do some testing, and here's how desktop screenreaders handle `aria-rowcount="-1"`:
- NVDA: reads that there are -1 rows directly, and gives no other hint about what that means
- Narrator: reads that there are 0 rows, regardless of how many rows are actually present
- JAWS: ignores the `aria-rowcount` attribute entirely, and reads the number of rows actually present (1, in the case of ShimmeredDetailsList)
- VoiceOver on macOS: reads that there are -1 rows, but gives no other hint about what that means

I don't think any of those behaviors are helpful, especially since our ShimmeredDetailsList does still expose the column header row. If we have `aria-rowcount="-1"` + that header row, we end up with weird behavior, like Narrator's "row 1 of 0". If support changes (not holding my breath in this case), we can always revisit. The risk of going against the ARIA spec recommendation here is extremely small -- I actually can't think of a single way updating the rowcount to reflect the header row would cause problems.

`aria-busy` is somewhat tangential here. It has very mixed support -- e.g. NVDA says "busy", Narrator does nothing, and it makes JAWS fully skip the grid as if it weren't there. We could potentially revisit `aria-busy` vs. an actual live region loading/loaded announcement, but since that requires localized strings, it seems like a better fit for v9.